asttokens/util.py

```diff
@@ -124,30 +140,36 @@ def iter_children_ast(node):
 # These feel hacky compared to isinstance() but allow us to work with both ast and astroid nodes
 # in the same way, and without even importing astroid.
 def is_expr(node):
+  # type: (AstNode) -> bool
   """Returns whether node is an expression node."""
   return node.__class__.__name__ in expr_class_names
 
 def is_stmt(node):
+  # type: (AstNode) -> bool
   """Returns whether node is a statement node."""
   return node.__class__.__name__ in stmt_class_names
 
 def is_module(node):
+  # type: (AstNode) -> bool
   """Returns whether node is a module node."""
   return node.__class__.__name__ == 'Module'
 
 def is_joined_str(node):
+  # type: (AstNode) -> bool
   """Returns whether node is a JoinedStr node, used to represent f-strings."""
   # At the moment, nodes below JoinedStr have wrong line/col info, and trying to process them only
   # leads to errors.
   return node.__class__.__name__ == 'JoinedStr'
 
 
 def is_starred(node):
+  # type: (AstNode) -> bool
   """Returns whether node is a starred expression node."""
   return node.__class__.__name__ == 'Starred'
 
 
 def is_slice(node):
+  # type: (NodeNG) -> bool
```

Comment:
`.elts` is only on `NodeNG`, so can't use `ASTNode`

Reply:
It definitely exists in `ast`:

```
>>> ast.parse('1, 2').body[0].value.elts
[<_ast.Constant object at 0x7f56dd8ba5b0>, <_ast.Constant object at 0x7f56dd8ba3d0>]
```

Maybe you need to specify specific types of node?